couchbase_cluster_admin/cluster.py

```diff
@@ -85,6 +85,25 @@ def enable_services(self):
         if resp.status_code != 200:
             raise Exception(f"Failed to enable services: {resp.text}")
 
+    def get_cluster_nodes(self):
```

Comment:
This copies most of the self.pool_info() property. It could indeed return `self.pool_info["nodes"]` to make it simpler.

There's also a vaguely named self.known_nodes() property that returns otpNode of each node.

Reply:
Thanks, I had not seen those. Will rework this then.